src/log/Log.cc

```diff
@@ -178,6 +179,26 @@ void Log::stop_graylog()
   m_graylog.reset();
 }
 
+void Log::set_journald_level(int log, int crash)
+{
+  std::scoped_lock lock(m_flush_mutex);
+  m_journald_log = log;
+  m_journald_crash = crash;
+}
+
+void Log::start_journald()
+{
+  std::scoped_lock lock(m_flush_mutex);
+  if (! m_journald.get())
+    m_journald = std::make_shared<Journald>(m_subs);
```

Comment:
why use a shared pointer? i think `unique_ptr<>` is good enough here.

Reply:
This is also copied from Graylog.